configs/components/fesom/fesom-2.1.yaml

```diff
@@ -452,6 +503,8 @@ config_files:
         io:      io
         cvmix:   cvmix
         icepack: icepack
+        tra:     tra
+        dyn:     dyn
```

Comment:
Relevant only for version 2.5. Can we place them under the choose block for the version?